Peddler.lua

```diff
@@ -548,7 +548,7 @@ local function markWares()
 	end
 end
 
-ns.markWares = markWares
```

Comment:
What did `ns` mean? Look like you also have a better namespace now :)!

Reply:
I suppose `ns` stands for `namespace`?  😅